src/main/java/ledger/user_interface/ui_controllers/AccountPopupController.java

```diff
@@ -72,6 +73,7 @@ private void insertDone() {
     }
 
     private void insertFail(Exception e) {
+        this.setupErrorPopup("Account insertion error.");
```

Comment:
Should probably keep the message. Latter definitely want to translate it but even the bare message should be useful.